src/test/java/chat/teco/tecochat/chat/domain/chat/ChatTest.java

```diff
@@ -151,4 +152,55 @@ class ChatTest {
             assertThat(chat.title()).isEqualTo("변경 제목");
         }
     }
+
+    @Nested
+    class 채팅_복사_시 {
+
+        private Chat chat;
+
+        @BeforeEach
+        void setUp() {
+            chat = new Chat(GPT_4, BACK_END_SETTING, "제목", 1L);
+            chat.addQuestionAndAnswer(new QuestionAndAnswer("질문1", "답변1", 10));
+            chat.addQuestionAndAnswer(new QuestionAndAnswer("질문2", "답변2", 20));
+            chat.increaseLike();
+            chat.increaseLike();
+            chat.increaseComment();
+        }
+
+        @Test
+        void 진행한_채팅을_모두_복사한다() {
```

Comment:
```suggestion
        void 진행한_채팅을_모두_복사한다_이때_좋아요_수와_댓글_수는_복사되지_않는다() {
```
로 변경하고 아래 주석을 지우는건 어때?

Reply:
오 좋은데요?